packages/headless/src/api/search/search/product-recommendation.ts

```diff
@@ -2,69 +2,89 @@ export interface ProductRecommendation {
   /**
    * The SKU of the product.
    */
-  sku: string;
+  permanentid: string;
+  /**
+   * A direct link to the product in URL format.
+   */
+  clickUri: string;
   /**
    * Name of the product.
    *
    * From the `ec_name` field.
    */
-  name: string;
-  /**
-   * A direct link to the product in URL format.
-   */
-  link: string;
+  ec_name: string;
   /**
    * Product brand.
    *
    * From the `ec_brand` field.
    */
-  brand?: string;
+  ec_brand?: string;
   /**
    * Category of the product (e.g., `"Electronics"`, `"Electronics|Televisions"`, or `"Electronics|Televisions|4K Televisions"`)
    *
    * From the `ec_category` field.
    */
-  category?: string;
+  ec_category?: string;
   /**
    * Base price of the product or variant.
    *
    * From the `ec_price` field.
    */
-  price?: number;
+  ec_price?: number;
   /**
    * Short description of the product.
    *
    * From the `ec_shortdesc` field.
    */
-  shortDescription?: string;
+  ec_shortdesc?: string;
   /**
-   * Product image in URL format.
+   * Product images in URL format.
    *
-   * From the `ec_thumbnail` field.
+   * From the `ec_thumbnails` field.
    */
-  thumbnailUrl?: string;
+  ec_thumbnails?: string;
   /**
    * Additional product images in URL format.
    *
    * From the `ec_images` field.
    */
-  imageUrls?: string[];
+  ec_images?: string[];
   /**
    * Promotional price of product or variant.
    *
    * From the `ec_promo_price` field.
    */
-  promoPrice?: number;
+  ec_promo_price?: number;
   /**
    * Availability of the product (i.e., whether the product is in stock).
    *
    * From the `ec_in_stock` field.
    */
-  inStock?: boolean;
+  ec_in_stock?: boolean;
   /**
    * A rating based system from 0-10.
    *
    * From the `ec_rating` field.
    */
-  rating?: number;
+  ec_rating?: number;
+
+  /**
+   * An object containing the requested additional fields for the product.
+   */
+  additionalFields: Record<string, unknown>;
 }
+
+export const ProductRecommendationDefaultFields: string[] = [
```

Comment:
I think it makes sense to put this here. If we change the `ProductRecommendation` interface, we should also change the default fields

Reply:
Why not place it inside the function building the request?

Reply:
When two things need to change together, I like to put them physically close to each other.